### PR TITLE
Bump to version 0.4.0

### PR DESCRIPTION
**Pull request by jakirkham**

### CODE REVIEW

**Comment by jakirkham** [comment]
@samuelgarcia @apdavison, seem to be running into an issue trying to release 0.4.0. Though it isn't making a lot of sense to me. The error message is shown below. More details can be seen in this [log](https://travis-ci.org/conda-forge/python-neo-feedstock/jobs/148318729#L739). If I look at the git tag [0.4.0](https://github.com/NeuralEnsemble/python-neo/tree/0.4.0/neo/io), I don't actually see `hdf5io_new.py`. Though it does appear in the PyPI sdist. In fact, I don't even see this file in `master`, which was at commit ( https://github.com/NeuralEnsemble/python-neo/commit/fca866b330518273b6122a72d9c1aa0a7439ee8e ) when writing this. In fact, it seems to be coming from this PR ( https://github.com/NeuralEnsemble/python-neo/pull/249 ) I believe. Is it alright if I switch to the git tag to do this release instead? I know PyPI doesn't allow people to upload a new sdist for the same release.

```
Compiling /Users/travis/miniconda3/envs/_build/lib/python2.7/site-packages/neo/io/hdf5io_new.py ...

Sorry: IndentationError: expected an indented block (hdf5io_new.py, line 49)

Command failed: /Users/travis/miniconda3/envs/_build/bin/python -Wi /Users/travis/miniconda3/envs/_build/lib/python2.7/compileall.py -q -x port_v3 /Users/travis/miniconda3/envs/_build/lib/python2.7/site-packages
```

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 
 package:
   name: python-{{ name }}
   version: {{ version }}
 
 source:
   fn: python-{{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  md5: 98e7e8948158f4492fc94d9d44367c91
+  url: https://github.com/NeuralEnsemble/python-neo/archive/{{ version }}.tar.gz
```
Comment (jakirkham) [inline recipe/meta.yaml:10]: I've gone ahead and switched to the git tag. If it is ok with both of you ( @apdavison @samuelgarcia ), please feel free to merge.

**Comment by apdavison** [comment]
This is my fault, I built the sdist in my working directory without cleaning up work-in-progress. I've now released neo-0.4.1 on PyPI with a clean sdist.

**Comment by jakirkham** [comment]
No worries. Thanks for cutting a fresh release. I'm adding it in PR ( https://github.com/conda-forge/python-neo-feedstock/pull/3 ).

**Pull request closed without merging**